lib/common/resources/neuron/pytorch-inference-resnet50.yml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Move this file outside of repo!

Reply:
would this be a good location: https://github.com/aws-observability/aws-observability-accelerator/tree/main/artifacts/k8s-deployment-manifest-templates ?

Reply:
Works for me!

Reply:
create a new folder like `neuron`

Reply:
done, see pr https://github.com/aws-observability/aws-observability-accelerator/pull/39